build.gradle

```diff
@@ -32,5 +39,41 @@ dependencies {
     compile('org.springframework.boot:spring-boot-starter-thymeleaf')
     compile('org.junit.jupiter:junit-jupiter-api')
     compile('org.springframework.boot:spring-boot-starter-security')
-    compile group: 'org.simplejavamail', name: 'simple-java-mail', version: '4.1.3'
+    compile('org.simplejavamail:simple-java-mail:4.1.3')
 }
+
+
+task beanstalkArchive(type: Zip, dependsOn: jar) {
+    from ('src/main/eb') {
```

Comment:
Don't you need to add folder "src/main/eb" and content to version control?
Wasn't there some files like env.config or something like that that need to be added?
How do you set the java command line options?